oak_functions/loader/src/server.rs

```diff
@@ -680,3 +695,67 @@ pub fn channel_create() -> (Endpoint, Endpoint) {
     };
     (endpoint0, endpoint1)
 }
+
+#[allow(dead_code)]
+struct RuntimeEndpoints {
+    lookup: Endpoint,
+}
+struct ChannelSwitchboard(HashMap<ChannelHandle, Arc<Endpoint>>);
```

Comment:
BTW does the endpoint really need to be in an Arc? It seems that the hashmap can just own the endpoint outright. Anyways we can iterate on it, but I'm curious if there is already a use case that requires Arcs. Maybe channel_wait?

Reply:
That is right, `Arc` is a relic here. I removed it.